common/persistence/nosql/nosqlplugin/cassandra/gocql/public/testBase.go

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2017-2020 Uber Technologies, Inc.
+// Copyright (c) 2017-2021 Uber Technologies, Inc.
+// Portions of the Software are attributed to Copyright (c) 2020 Temporal Technologies Inc.
```

Comment:
oh, temporal has the same structure for those tests?

Reply:
No I didn’t look at any temporal code for this refactoring. Moving the files is because of circle dependency.

Reply:
then we can remove the second line?

Reply:
ah I see. Yeah I copied it from somewhere else by mistake.